include/RAJA/policy/cuda/reduce.hpp

```diff
@@ -60,23 +66,23 @@ template <typename T>
 struct atomic<sum<T>> {
   RAJA_DEVICE RAJA_INLINE void operator()(T& val, const T v)
   {
-    RAJA::atomicAdd<T>(RAJA::cuda_atomic{}, &val, v);
+    RAJA::atomicAdd(RAJA::cuda_atomic{}, &val, v);
```

Comment:
Should we include the template parameter? e.g. RAJA::atomicAdd<T>

Reply:
Hmm, I think it might interfere with the templated types in desul/atomic.hpp since it would be ``RAJA::atomicAdd<policy, T>(...)`` and there are no defaulted types as of now. I might need to double-check that.

Reply:
From testing I haven't found any issues one way or the other, so it might just be worth keeping the ``<T>`` in since removing it doesn't seem to do much. I could've sworn there was some issue with it at some point though, but that might have just been an artifact of a different bugfix.

Reply:
If it isn't necessary, I'd leave it out and let the compiler infer it from the arguments.  One less thing to worry someone will accidentally break later by mucking with T or the way the argument declarations are written.